src/taco-cli/cli/utils/kitHelper.ts

```diff
@@ -102,7 +102,8 @@ class KitHelper {
 
     private static acquireKitPackage(): Q.Promise<ITacoKits> {
         if (!KitHelper.kitPackagePromise) {
-            KitHelper.kitPackagePromise = TacoPackageLoader.lazyTacoRequire<ITacoKits>(KitHelper.TACO_KITS_NPM_PACKAGE_NAME, KitHelper.dynamicDependenciesLocation, tacoUtility.InstallLogLevel.taco);
+            KitHelper.kitPackagePromise = TacoPackageLoader.lazyTacoRequire<ITacoKits>(KitHelper.TACO_KITS_NPM_PACKAGE_NAME,
+                KitHelper.dynamicDependenciesLocation, tacoUtility.InstallLogLevel.error);
```

Comment:
Thankfully they do check for >= :)

$ grep -r '= InstallLogLevel.taco' .
./taco-utils/buildInfo.ts:                    this.logLevel = InstallLogLevel.taco;
./taco-utils/tacoPackageLoader.ts:            if (typeof logLevel !== "undefined" && logLevel !== InstallLogLevel.taco) {
./taco-utils/tacoPackageLoader.ts:            if (request.logLevel >= InstallLogLevel.taco) {
./taco-utils/tacoPackageLoader.ts:                    if (request.logLevel >= InstallLogLevel.taco) {
./taco-utils/tacoPackageLoader.ts:                        if (request.logLevel >= InstallLogLevel.taco) {